modules/core-user-password.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="core-user-password_{context}"]
+= Changing the core user password for node access
+
+By default, {op-system-first} creates a user named `core` on the nodes. In the event that you cannot use SSH or the `oc debug node` command to access a node, you can use the `core` user to access the node through a cloud provider serial console or a bare metal baseboard controller manager (BMC). However, by default, there is no password for this user, so you cannot log in. 
+
+You can create a password the `core` user by using use a machine config. The Machine Config Operator (MCO) assigns the password and injects the password into the `/etc/shadow` file, allowing you to log in with the `core` user. The MCO does not examine the password hash. Thus, the MCO cannot report if there is a problem with the password.
+
+[NOTE]
+====
+* The password works only through a cloud provider serial console or a BMC. It does not work with SSH.
+
+* If you have a machine config that includes a systemd unit that sets a password or adds a file to the `/etc/shadow` file, it takes precedence over the password hash.
+====
+
+You can change the password, if needed, by editing the machine config you used to create the password. Also, you can remove the password by deleting the machine config. Deleting the machine config does not remove the user account. 
+
+.Prerequisites
+
+* Create a hashed password by using a tool that is supported by your operating system.
+
+.Procedure
+
+. Create a machine config that contains the `core` username and the hashed password:
++
+[source,terminal]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 99-worker-password
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    passwd:
+      users:
+      - name: core <1>
+        passwordHash: $6$2sE/010goDuRSxxv$o18K52wor.wIwZp <2>
+----
+<1> This must be `core`.
+<2> The hashed password to use with this user account.
+
+. Create the machine config:
++
+[source,yaml]
+----
+$ oc create -f <file-name>.yaml
+----
++
+The nodes takes a few moments to become available because they do not reboot. You can use the `oc get mcp` to watch for the machine config pools to be updated:
```

Comment:
```suggestion
The nodes take a few moments to become available because they do not reboot. You can use the `oc get mcp` to watch for the machine config pools to be updated:
```